src/js/node/child_process.ts

```diff
@@ -1359,9 +1360,17 @@ function nodeToBun(item, index) {
   if (typeof item === "number") {
     return item;
   }
+  if (item instanceof StreamModule.Readable) {
+    if (Object.hasOwn(item, "fd") && typeof item.fd === "number" && item.fd >= 0 && item.fd <= 2) return item.fd;
+    throw new Error(`TODO: stream.Readable stdio @ ${index}`);
+  }
+  if (item instanceof StreamModule.Writable) {
```

Comment:
Looks like we can check for `write` and `on` functions

https://github.com/oven-sh/bun/blob/e6954c440e03d27c3b56d13a7834c2b7dfbb3123/src/js/node/stream.ts#L836-L838